src/integrations/ems-Slack-Bridge.md

```diff
@@ -29,3 +29,10 @@ Once reprovisioning is finished, you are able to bridge Slack to your EMS server
 ## Setup
 
 The setup process for the EMS Slack bridge is explained [here](./morg-Slack-Bridge.md)
+
+## Initiate a DM with a Slack user from Matrix
+
+- Message the Slack bot user on Matrix
```

Comment:
```suggestion
- Message the Slack bot user on Matrix.
```
This nitpick feedback :-) is because I think we agreed to end sentences with a full stop unless there was an image after the sentence?